<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScene</class>
 <widget class="QDialog" name="LoginScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widgetTitle" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledt_userName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>180</width>
     <height>38</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="placeholderText">
    <string>请输入账号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledt_passwd">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>180</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_login">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>登  录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbox_remember">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbox_autoLogin">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbe_login_error">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>错误提示</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
